app/controllers/HomeController.scala

```diff
@@ -128,6 +130,8 @@ class HomeController @Inject()(val homeView: views.html.Home,
 
       val accountSettingsTileViewModel = AccountSettingsTileViewModel(currentTaxYear, isEnabled(ReportingFrequencyPage), currentITSAStatus)
 
+      val penaltiesAndAppealsTileViewModel = PenaltiesAndAppealsTileViewModel(maybePenaltyPoints)
```

Comment:
should be fine to return a viewModel right from the service, but it is really your choice how you like to implement this